server/pulp/server/async/celery_instance.py

```diff
@@ -90,5 +90,7 @@ def configure_SSL():
         }
         celery.conf.update(BROKER_USE_SSL=BROKER_USE_SSL)
 
+        if config.has_option('tasks', 'broker_login_method'):
```

Comment:
Once the default gets added this check will need to check if the user has set it differently then the default.